index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microverse Pair Programming Tribute Project</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div id="main">
+      <!-- header -->
+      <div class="header-container myflex">
+        <div class="head-content">
+          <h1 id="title">President Buhari</h1>
+        </div>
+      </div>
+      <div class="img-container" id="img-div">
+        <img src="nj.jpeg" alt="" style="width: 100%" />
+        <!-- caption here -->
+        <div class="img-cap" id="img-caption">
+          <p>Nigerian's President Buhari and his family</p>
+        </div>
+      </div>
+      <!-- trubute info starts here -->
+      <section id="tribute-info">
+        <h1>Here is a time line of President Buhari's Life</h1>
+        <div class="content">
+          <ul>
+            <p>
+              Muhammadu Buhari GCFR is a Nigerian politician who has been
+              President of Nigeria since 2015. Buhari is a retired Nigerian Army
+              Major General and has served as military head of state in Nigeria
+              between 1983 to 1985, after taking power in a military coup
+              d'état.
+            </p>
+            <li>
```

Comment:
- [ ] [OPTIONAL] The `p` tag is not an element of `ul`, consider taking it out of the `ul` tag